CHANGELOG.md

```diff
@@ -0,0 +1,99 @@
+- Disallow Z designators when parsing strings for Plain Temporal types ([f3f8a994], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1874))
+- Allow ISO strings with "Z" + a bracketed IANA name ([70bd9898], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1749))
+- Emit ES2020 builds for newer browsers, and emit ES5 for older browsers. ([2331468d], [9e95c62b])
+- Temporal.Duration constructor will now throw if given a non-integer ([9df5d068], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1872))
+- Remove support for sub-minute offsets in ISO strings ([766e5037], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1871), [Spec PR](https://github.com/tc39/proposal-temporal/pull/1862))
+- Throw TypeError on missing options from Duration.total ([4ec075f0], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1720))
+- Reject non-integer Duration fields in Duration.with() ([e6b2488d], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1735))
+- Ensure an Object is returned from calendar.mergeFields() ([4e63f25f], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1719))
+
+Bug fixes:
+
+- Fix GetFormatterParts for Firefox Nightly ([47f9132f])
+- Fix TS types of RoundTo and TotalOf ([3008a670])
+- Fix crash setting `day` outside current JPN era ([6d3588c3], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1807))
+- Copy options object for PlainYearMonth.{add,subtract} and InterpretTemporalDateTimeFields to prevent user-modified objects from interfering with later operations. ([bafa1bdf], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1748))
+- Validate input to Calendar.prototype.fields ([7ebc700e], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1750))
+- Stop observably calling into `getPossibleInstantsFor` in `InterpretISODateTimeOffset` ([5448e59f], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1688))
+- Call `CalendarEquals` correctly ([07ea694e], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1858))
+- Fix arithmetic issues when using non-ISO months ([079a3325], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1761))
+
+Non-breaking changes:
+
+- Various `#round` and `#total` methods now accept string parameters or options bags. Strings are interpreted as the `smallestUnit` option (or `unit` for `Temporal.Duration#total`). ([068e801f], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1875))
+- Add @@toStringTag to TS types ([41ab6bc0])
+- Accept string Calendar names in PlainMonthDay and PlainYearMonth constructors ([27b4c7e8])
+- Make options optional in Calendar method TS types ([3a09d00d])
+- Regex: tighten matching of month and day values in datesplit ([b5736546], see also [Spec PR](https://github.com/tc39/proposal-temporal/pull/1836))
```

Comment:
Is this really a non-breaking change?  Given that it introduces new exceptions for cases that didn't throw before, is it a bug fix or a breaking change?

Reply:
Agreed this is actually a bug fix, not a non-breaking change. Moved.